likecoin/likecoin.php

```diff
@@ -447,7 +449,7 @@ function likecoin_add_privacy_policy_content() {
 		__(
 			'When you use the LikeCoin embed, we automatically collect basic visitor informations,
 		e.g. IP address, user agent, etc. These kind of information might be used as analytics purpose.
-		For the purpose of applying personal site preferences, We might also identify a registered 
+		For the purpose of applying personal site preferences, We might also identify a registered
```

Comment:
although this space is quite questionable, probably don't want to change it in this commit